setup.py

```diff
@@ -78,7 +78,7 @@ def package_files(prefix: str, directory: str, extension: str):
     'torchmetrics>=0.7.0,<0.8',
     'torch_optimizer>=0.1.0,<0.2',
     'torchvision>=0.10.0',  # torchvision has strict pytorch requirements
-    'torch>=1.10,<2',
+    'torch>=1.9,<2',
```

Comment:
We recently removed support for torch 1.9, is there a reason we need to re-introduce here?

Reply:
the code was segfaulting on the interactive tpu instance and this is the change that fixed it. I can look into there might be wrong interactions and will revert this for now.

Reply:
OK, but does that mean for TPU, the user needs to install `torch==1.9`? and that newer versions of torch are not supported?